Q 24) A Government  company claims that an average light bulb lasts 270 days. A researcher randomly selects 18 bulbs for testing. The sampled bulbs last an average of 260 days, with a standard deviation of 90 days. If the CEO's claim were true, what is the probability that 18 randomly selected bulbs would have an average life of no more than 260 days

Hint:  
rcode -> pt(tscore,df)  
df -> degrees of freedom


In [1]:
import scipy.stats as stats
from math import sqrt

In [2]:
# Given data
claim_mean = 270  # Claimed average lifespan
sample_mean = 260  # Sampled average lifespan
sample_stddev = 90  # Standard deviation of the sample
sample_size = 18  # Sample size

# Calculate the standard error of the mean (SEM)
sem = sample_stddev / sqrt(sample_size)

# Calculate the t-score
t_score = (sample_mean - claim_mean) / sem

# Degrees of freedom
df = sample_size - 1

# Calculate the probability using the cumulative distribution function (CDF)
probability = stats.t.cdf(t_score, df)

# Round the probability to two decimal places
rounded_probability = round(probability, 2)

# Display the rounded probability
rounded_probability

0.32

Answer (Q24): There is a 32% probability that 18 randomly selected bulbs would have an average life of no more than 260 days